onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -87,27 +87,95 @@ TEST(ResizeOpTest, ResizeOpLinearDownSampleTest_4DBilinear) {
   test.Run();
 }
 
+// Since NNAPI only using the scale calulate using the input/output size
+// So the 0.6 scale in above test will be treated as 0.5 in NNAPI
+// Add the following 2 tests to test with scales valid to NNAPI
```

Comment:
I don't quite understand this comment. If the 0.6 scale in the above test is treated as 0.5 in NNAPI isn't that incorrect?

Reply:
NNAPI or TFLIte's bilinear resize's implementation is a bit different than 
Update the comments, the test case is a bit extreme, since we use a very small image size (1,1,2,4), so the result is quite different, usually we can get within tolerant for larger images

---
In reply to: [502980925](https://github.com/microsoft/onnxruntime/pull/5427#discussion_r502980925) [](ancestors = 502980925)